docs/TEAM.md

```diff
@@ -1,6 +1,6 @@
 ## Beginner
 | Image | Name       | Description |
-| ---------- | ---------- | ----------- |
```

Comment:
Please keep this line, as this is required for the **table** format.

Reply:
> Please keep this line, as this is required for the **table** format.

Ok

Reply:
This issue is still not fixed. You just need to add

```md
| ---------- | ---------- | ----------- |
```

below this:

```md
| Image | Name       | Description |
```